lib/core/src/lib/form/components/widgets/core/form-field-validator.ts

```diff
@@ -583,3 +584,39 @@ export const FORM_FIELD_VALIDATORS = [
     new MinDateTimeFieldValidator(),
     new MaxDateTimeFieldValidator()
 ];
+
+export const momentToDateFnsMap = {
```

Comment:
you are using this maps in 2 places, second is translation service; extract to a separate class and use same import in both cases

Reply:
moved all logic to DateFnsUtils